priv/repo/migrations/20210421162709_create_part_component_registrations.exs

```diff
@@ -0,0 +1,51 @@
+defmodule Oli.Repo.Migrations.CreatePartComponentRegistrations do
+  use Ecto.Migration
+
+  def change do
+    create table(:part_component_registrations) do
+      add :slug, :string
+      add :title, :string
+      add :icon, :string
+      add :description, :text
+      add :delivery_element, :string
+      add :authoring_element, :string
+      add :delivery_script, :text
+      add :authoring_script, :text
+      add :allow_client_evaluation, :boolean
```

Comment:
Allow client evaluation is probably unique to Activities and doesn't apply to Part Components

Reply:
yeah I almost removed that, but then thought maybe it would be needed for something... I'll remove it though, better to add something *when* it is needed :)